jb-lib/lib/data_models.py

```diff
@@ -139,6 +140,8 @@ class FSMOutput(BaseModel):
     menu_title: Optional[str] = None
     form_token: Optional[str] = None
     plugin_uuid: Optional[str] = None
+    retriever_collection_name: str = "KB_Law_Files"
+    retriever_chunk_size: int = 5
```

Comment:
`Let's not call it chunk_size. Let's use `top_k`. It's better understood.

Reply:
Sure, I will update it.